blueocean-config/pom.xml

```diff
@@ -19,9 +19,13 @@
       <groupId>${project.groupId}</groupId>
       <artifactId>blueocean-web</artifactId>
     </dependency>
-      <dependency>
-          <groupId>${project.groupId}</groupId>
-          <artifactId>blueocean-commons</artifactId>
-      </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>blueocean-rest-impl</artifactId>
```

Comment:
Chatted with @michaelneale and his comment was wrt to my last comment about the seeming need to use `blueocean-rest-impl` instead of just `blueocean-rest`. I'll touch base with @vivek in case there's some CDI magic that needs to be used.